portal/build.sbt

```diff
@@ -0,0 +1,5 @@
+enablePlugins(JavaServerAppPackaging)
+mappings in Universal ++= Seq(
+  (baseDirectory in Compile).value / "src/main/resources/application.conf" -> "conf/application.conf",
+  (baseDirectory in Compile).value / "src/main/resources/logback.xml" -> "conf/logback.xml"
```

Comment:
`resourceDirectory` will be more appropriate here than `baseDirectory`.

Reply:
Exactly!